locale/locales/en.json

```
@@ -31,6 +31,14 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.RowsPerPageChange": {
+    "value": "Rows per page has changed. Now showing {0} rows on each page.",
+    "comment": "Label for rows per page change."
+  },
+  "SNTable.Pagination.PageStatusReport": {
+    "value": "You are currently on page {0} out of {1} pages.",
```

I think this one should be similar to the rowsPerPage one, maybe
```suggestion
    "value": "Page has changed. Now showing page {0} out of {1}.",
```

---

✅ Fixed